src/Razor/Microsoft.AspNetCore.Razor.Language/src/BoundAttributeDescriptorComparer.cs

```diff
@@ -60,6 +60,8 @@ public int GetHashCode(BoundAttributeDescriptor descriptor)
             hash.Add(descriptor.Kind, StringComparer.Ordinal);
             hash.Add(descriptor.Name, StringComparer.Ordinal);
             hash.Add(descriptor.IsEditorRequired);
+            hash.Add(descriptor.TypeName, StringComparer.Ordinal);
+            hash.Add(descriptor.Documentation, StringComparer.Ordinal);
```

Comment:
Hmmm, I'm not sure we should be hashing the documentation. The docs can be quite huge and computing a hash for it may not be cheap.

Reply:
@NTaylorMullen the problem is if we don't hash Documentation then the user won't get an updated Hover when the documentation is changed. As @pranavkm said we really ought to just stop trying to use hash as a key (Hash collisions could cause WILD behavior), but I needed something that would work while I came up with a better system.

Reply:
> @NTaylorMullen the problem is if we don't hash Documentation then the user won't get an updated Hover when the documentation is changed.

Fair enough; however, I do worry quite a bit on the perf implications of this